cvpcases/README.md

```diff
@@ -0,0 +1,26 @@
+This folder include all cvp cases and the other configuration to access a cluster will be generated by the jenkins job. 
+<br>
+#The case scripts
+<br>
+1) The script must be a shell using same name as the case name
+<br>
+2) The script should return pass( 0) or fail (not 0)
+<br>
+3) It is better to use relative path in your scripts
+<br>
+4) The script will be executed in this director {$PWD=this directory}
+
+#The following files will be placed under this directory by jenkins job.
+<br>
+1) The Build Artifacts from Launch Environment Flexy job  
+<br>
+    For example: https://openshift-qe-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/Launch%20Environment%20Flexy/57679/
```

Comment:
This is public repo,please don't share any internal IP  here.

Reply:
@anpingli I would also suggest that you move these helper script into `verification-tests` or `openshift-tests` repo

Reply:
@pruan-rht  That is a good idea to move to openshift-test repo.